static/src/js/components/OrderOptionList/OrderOptionList.jsx

```diff
@@ -92,7 +175,7 @@ const OrderOptionList = () => {
       dataKey: 'deprecated',
       title: 'Deprecated?',
       className: 'col-auto',
-      dataAccessorFn: (cellData) => (cellData || 'false')
+      dataAccessorFn: (cellData) => (cellData ? 'True' : 'False')
```

Comment:
Would love for this to be a checkmark icon or something instead of a string but its probably fine for now